packages/web/src/hooks/swap/use-swap-handler.tsx

```diff
@@ -33,9 +33,11 @@ const findKeyByValue = (
 
 export const useSwapHandler = () => {
```

Comment:
Could you please remove the change of use-swap-handler?
I covered snack bars and various contents in other PRs.

Reply:
I will preserve it